agent_app_sample_code/cookbook_utils/cookbook_config.py

```diff
@@ -0,0 +1,203 @@
+from pydantic import BaseModel, Field, root_validator, computed_field, field_validator, FieldValidationInfo
+import os
+import json
+import mlflow
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors.platform import ResourceAlreadyExists, ResourceDoesNotExist, NotFound, PermissionDenied
+
+class AgentCookbookConfig(BaseModel):
```

Comment:
Pydantic settings might get us more out of the box here: https://docs.pydantic.dev/latest/concepts/pydantic_settings/#usage